pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -183,7 +186,12 @@ static __always_inline int handle_retransmit(struct sock *sk, int segs) {
     }
 
     tcp_stats_t stats = { .retransmits = segs, .rtt = 0, .rtt_var = 0 };
-    update_tcp_stats(&t, stats);
+    if (retrans_type == RETRANSMIT_COUNT_ABSOLUTE) {
```

Comment:
This should be necessary, and won't actually work for the prebuilt version since you're referencing the `retrans_out` field directly. You should move this code to the runtime compiled version and pass the count in to this function.